build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+set -x
+TAG=`date +%Y-%m-%d`
+REPO=$1
+cp -R third_party/docker/* mkdebootstrap/
+cd mkdebootstrap && gcloud alpha container builds create . -t $REPO/mkdebootstrap:jessie
```

Comment:
Or in this case I think you can pass mkdebootstrap/ directly to the gcloud command, and avoid the "cd"ing